### ISSUE TITLE
The binding for .NET don't have the full support for iOS/Android

### ISSUE THREAD

**Issue by mamcx** [xamarin/flex#43]
I can pull the library for nuget, but need to manually integrate the code on the samples folder to make it work with iOS/Android.

**Comment by lrz** [comment]
What sample code are you referring to?

**Comment by mamcx** [comment]
The samples from https://github.com/xamarin/flex/tree/master/bindings/csharp/samples. I pull from the android, shared and iOS

**Comment by lrz** [comment]
Yeah but you said you had to integrate "code from the samples", what code in particular? As far as I know once you integrate the nuget its C# API should be available in your project, iOS or Android.
